aleth/MinerAux.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <boost/algorithm/string/case_conv.hpp>
+
+#include <libdevcore/CommonJS.h>
+#include <libethcore/BasicAuthority.h>
+#include <libethashseal/Ethash.h>
+#include <libethashseal/EthashCPUMiner.h>
+
+#include "MinerAux.h"
+
+using namespace std;
+using namespace boost::program_options;
+using namespace dev;
+using namespace eth;
+using namespace dev::eth;
+
+MinerCLI::MinerCLI(OperationMode _mode): mode(_mode)
+{
+    Ethash::init();
```

Comment:
Didn't you remove these in the last PR?

Reply:
Yes, I forgot it, sorry :)

Regarding order of `#include`, does it put local includes on top?

Next time I'll rebase to clean up commits, I'm still new to this repo :). Thanks for resolving conflicts.

Reply:
It will put local includes first, with `"MinerAux.h"` on top. If you want additional grouping, separate groups with empty line.

Reply:
@gumb0 @chfast I removed `MinerCLI` ctor, it's empty now. `clang-format` doesn't re-organize `#include` but I also updated them manually.